NSS-Alleged-Donkeys/WorkforceManagement
# JE1-CreateDepartment

## Link to Ticket
closes #1 [Link](https://github.com/NSS-Alleged-Donkeys/WorkforceManagement/issues/1)

## Description of Proposed Changes
Wrote test to make sure HTML of Create Department Form exists when the view is loaded

## Steps to Test
Run ```Get_CreateDepartmentForm()``` inside of DepartmentTest.cs
Make sure test is successful


## Impacted Areas in Application
DepartmentTest.cs
